extensions/amp-story/amp-story-animations.md

```diff
@@ -0,0 +1,214 @@
+<!---
```

Comment:
We should maybe keep this one in the `<amp-story>` reference doc for now since it isn't it's own component. It might be worth linking to the section from the other docs since they can be animated as well. WDYT?